contracts/test/ProjectRegistry-test.ts

```diff
@@ -132,17 +174,36 @@ describe("ProjectRegistry", function () {
     expect(currentOwners[2]).to.equal(this.accounts[0].address);
     expect(currentOwners[3]).to.equal(this.owner.address);
 
-    await this.contract.connect(this.owner).removeProjectOwner(projectID, this.accounts[1].address, this.accounts[0].address)
+    await this.contract
+      .connect(this.owner)
+      .removeProjectOwner(
+        projectID,
+        this.accounts[1].address,
+        this.accounts[0].address
+      );
     expect(await this.contract.projectOwnersCount(projectID)).to.equal("3");
     let newOwners = await this.contract.getProjectOwners(projectID);
     expect(newOwners.length).to.equal(3);
 
-    await this.contract.connect(this.owner).removeProjectOwner(projectID, this.accounts[1].address, this.owner.address)
+    await this.contract
+      .connect(this.owner)
+      .removeProjectOwner(
+        projectID,
+        this.accounts[1].address,
+        this.owner.address
+      );
```

Comment:
@schultztimothy I'm not sure about all these lints fixes, in the end this line is 115 which should be good for displays now, vs having 7 lines. wdyt if we change the configuration or remove this check?

Reply:
Yeah looks like the prettier recommendations were taking precedent. We should be good now with https://github.com/gitcoinco/grants-hub/pull/104/commits/6236d09aaf0193011c9c671fc4bd71d15b3a1d2c